pom.xml

```
@@ -1,128 +1,125 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <name>Session Service</name>
-    <description>RESTful API to cBioPortal/cbioportal sessions in MongoDB.</description>
-    <url>https://github.com/cBioPortal/session-service/</url>
-    <groupId>org.cbioportal.session_service</groupId>
-    <artifactId>session_service</artifactId>
-    <version>0.3.0</version>
-    <packaging>${packaging.type}</packaging>
-
-    <!-- inherit defaults from spring boot -->
-    <parent>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <name>Session Service</name>
+  <description>RESTful API to cBioPortal/cbioportal sessions in MongoDB.</description>
+  <url>https://github.com/cBioPortal/session-service/</url>
+  <groupId>org.cbioportal.session_service</groupId>
+  <artifactId>session_service</artifactId>
+  <version>0.2.0</version>
```

@ao508 I don't think we want to downgrade: maybe `0.4.0` is better?
```suggestion
  <version>0.4.0</version>
```

---

ah I didnt catch that when merging the changes from the other 2 open PRs - updating now